packages/x-date-pickers/src/TimeField/tests/editing.TimeField.test.tsx

```diff
@@ -204,6 +204,250 @@ describe('<TimeField /> - Editing', () => {
     });
   });
 
+  describeAdapters('key: PageDown', TimeField, ({ adapter, testFieldKeyPress }) => {
+    describe('24 hours format (PageDown)', () => {
+      describe('Hours field', () => {
+        it('should set hours field to maximal when no default value is provided', () => {
+          testFieldKeyPress({
+            format: adapter.formats.hours24h,
+            key: 'PageDown',
+            expectedValue: '23',
+            selectedSection: 'hours',
+          });
+        });
+
+        it('should decrement hours field by 5 when default value is provided', () => {
+          testFieldKeyPress({
+            format: adapter.formats.hours24h,
+            key: 'PageDown',
+            defaultValue: adapter.date('2024-06-04T10:25:00'),
+            expectedValue: '05',
+            selectedSection: 'hours',
+          });
+        });
+
+        it('should trespass hours field when default value is lower than 5', () => {
```

Comment:
Nitpick: Given [the definition](https://www.merriam-webster.com/dictionary/trespass) of `trespass`, WDYT about going with `flip` for simplicity? 🤔 
This would apply to all instances.